x-pack/plugins/ingest_manager/public/applications/ingest_manager/layouts/default.tsx

```diff
@@ -19,6 +20,8 @@ interface Props {
 const Container = styled.div`
   min-height: calc(100vh - ${props => props.theme.eui.euiHeaderChildSize});
   background: ${props => props.theme.eui.euiColorEmptyShade};
+  display: flex;
+  flex-direction: column;
```

Comment:
I reverted the flex properties here, as well as in `with_header` and `without_header` layouts in https://github.com/elastic/kibana/pull/63992.

The flex properties were breaking other pages, causing them not to expand to full width. It didn't seem like the removal affected the overview page contents or the alpha messaging so hopefully it's good to go. For future reference if a page content needs flex, the individual pages should wrap its contents in EuiFlexGroup or similar. The base layouts will rarely need adjusting :)